pyproject.toml

```diff
@@ -13,9 +13,8 @@ repository = "https://github.com/trezor/python-slip10"
 keywords = ["bitcoin", "slip10", "hdwallet"]
 
 [tool.poetry.dependencies]
-cryptography = "*"
-ecdsa = "*"
-python = ">=3.8,<4.0"
+cryptography = ">=45"
+python = ">3.9.0,<3.9.1 || >3.9.1,<3.14.0"
```

Comment:
1. ISTM that `>3.9.0,<3.9.1` is not satisfiable, so only the latter condition makes sense. Or am I misunderstanding the syntax?
2. Curious, why is 3.14 excluded?
3. Let's update `.github/workflows/python-package.yml` accordingly.